packages/bumbag/src/Modal/Modal.tsx

```diff
@@ -1,14 +1,26 @@
-import * as React from 'react';
-import { Box as ReakitBox, DialogProps as ReakitDialogProps, useDialog as useReakitDialog } from 'reakit';
-import _merge from 'lodash/merge';
+import * as React from "react";
```

Comment:
Can we format this file? :)

Reply:
Done. One more thing though, this component errored out when trying to update the snapshot:
<img width="957" alt="Screen Shot 2021-03-14 at 21 57 02" src="https://user-images.githubusercontent.com/13311268/111106068-98d58380-8511-11eb-9779-8517957c9d6f.png">

I haven't looked too closely, but was wondering if you have a better idea of what's causing it